aviary/interface/methods_for_level2.py

```diff
@@ -2325,6 +2327,169 @@ def run_aviary_problem(self,
+        prob_alternate.add_driver(optimizer, max_iter=max_iter, verbosity=verbosity)
+        prob_alternate.add_design_variables()
+        prob_alternate.add_objective()
+        prob_alternate.setup()
+        prob_alternate.set_initial_guesses()
+        prob_alternate.run_aviary_problem(record_filename='alternate_problem_history.db')
+        return prob_alternate
+
+    def run_fallout_mission(self, json_filename='sizing_problem.json',
+                            mission_mass=None, payload_mass=None, phase_info=None,
+                            optimizer=None, max_iter=50, verbosity=Verbosity.BRIEF):
+        """
+        This function runs a fallout mission based on a sizing mission output.
+
+        Parameters
+        ----------
+        json_filename : str
+            Name of the file that the sizing mission has been saved to.
+        mission_mass : float, optional
+            Takeoff mass for the fallout mission.
+        payload_mass : float, optional
+            Mass of the payload for the mission.
+        phase_info : dict, optional
+            Dictionary containing the phases and their required parameters.
+        optimizer : str, optional
+            Optimizer to be used for solving the fallout mission.
+        max_iter : int, optional
+            Max number of iterations allowed for the optimizer.
+        verbosity : Verbosity or list, optional
+            If Verbosity.DEBUG, debug print options ['desvars','ln_cons','nl_cons','objs'] will be set.
+            If a list is provided, it will be used as the debug print options.
+        """
+        if phase_info is None:
+            phase_info = self.phase_info
+        if mission_mass is None:
+            mission_mass = self.get_val(Mission.Design.GROSS_MASS)
+        if payload_mass is None:
+            if self.mission_method is HEIGHT_ENERGY:
+                payload_mass = self.get_val(Aircraft.CrewPayload.TOTAL_PAYLOAD_MASS)
+            elif self.mission_method is TWO_DEGREES_OF_FREEDOM:
+                payload_mass = self.get_val(Aircraft.CrewPayload.PASSENGER_PAYLOAD_MASS)
+        if optimizer is None:
+            optimizer = self.driver.options["optimizer"]
+
+        design_range = self.get_val(Mission.Design.RANGE)
+
+        prob_fallout = _load_off_design(json_filename, ProblemType.FALLOUT, phase_info,
+                                        payload_mass, design_range, mission_mass)
+
+        prob_fallout.check_and_preprocess_inputs()
+        prob_fallout.add_pre_mission_systems()
+        prob_fallout.add_phases()
+        prob_fallout.add_post_mission_systems()
+        prob_fallout.link_phases()
+        prob_fallout.add_driver(optimizer, max_iter=max_iter, verbosity=verbosity)
+        prob_fallout.add_design_variables()
+        prob_fallout.add_objective()
+        prob_fallout.setup()
+        prob_fallout.set_initial_guesses()
+        prob_fallout.run_aviary_problem(record_filename='fallout_problem_history.db')
```

Comment:
I wonder if we should split this into 2 functions, where the first one "sets up" the model, doing everything up to, but not including, run_aviary_problem.  And then a second function to run the problem.  This is to allow the user to do any extra sets or to change optimizer settings. 

Note, I don't have a specific usecase for this -- just thinking that maybe some other variables might be different in off-design, particularly when we include external subsystems.